### PR TITLE
DeviceName and actions

### PR DESCRIPTION
**Pull request by benjaminSchilling33**
This pull request adds a new value to the discoverResponse message.
It allows to give the device a user defined name.

Additionally the discoverResponse message returns the actions the device can handle.

### CODE REVIEW

File: ScpDebug.cpp [hunk 1]
```diff
@@ -16,13 +16,23 @@ ScpDebug::ScpDebug()
 void ScpDebug::println(logClass_t logClass, String debugString)
 {
 #ifdef DEBUG
-    Serial.println(debugString);
+    String indent = "";
+    for (int i = 0; i < logClass; i++)
+    {
+        indent = indent += " ";
```
Comment (marceljaehn) [inline ScpDebug.cpp]:
```suggestion
        indent += " ";
```

File: ScpDebug.cpp [hunk 2]
```diff
@@ -16,13 +16,23 @@ ScpDebug::ScpDebug()
...
 }
 
 void ScpDebug::print(logClass_t logClass, String debugString)
 {
 #ifdef DEBUG
-    Serial.print(debugString);
+    String indent = "";
+    for (int i = 0; i < logClass; i++)
+    {
+        indent = indent += " ";
```
Comment (marceljaehn) [inline ScpDebug.cpp]:
```suggestion
        indent += " ";
```

File: ScpEepromController.cpp [hunk 3]
```diff
@@ -123,17 +122,17 @@ String ScpEepromController::getPassword()
...
    return password;
 }
 
 /**
    * @brief 
    * 
-   * @return password 
+   * @param password 
    */
 void ScpEepromController::setPassword(String password)
```
Comment (marceljaehn) [inline ScpEepromController.cpp:137]: currentPasswortNumber is given as return value. This should be reflected in the return type.
```suggestion
uint32_t ScpEepromController::setPassword(String password)
```

**Review by marceljaehn — CHANGES_REQUESTED** [review]

File: ScpEepromController.cpp (hunk 3 above)
Comment (benjaminSchilling33) [reply ScpEepromController.cpp:137]: I will note this as a future improvement.

**Review by marceljaehn — APPROVED** [review]

**Pull request merged**